backend/oqtopus_cloud/provider/routers/jobs.py

```diff
@@ -39,42 +40,73 @@
 
 @router.get(
     "/jobs",
-    response_model=list[JobDef],
+    response_model=list[Union[JobDef, GetJobsResponse]],
+    responses={500: {"model": Detail}},
 )
 @tracer.capture_method
 def get_jobs(
     device_id: str,
+    fields: Optional[str] = None,
     status: Optional[str] = None,
     max_results: Optional[int] = None,
     timestamp: Optional[str] = None,
     db: Session = Depends(get_db),
-) -> list[JobDef]:
+) -> list[Union[JobDef, GetJobsResponse]] | ErrorResponse:
```

Comment:
`Union` and `|` is same expression

Reply:
https://github.com/oqtopus-team/oqtopus-cloud/blob/95b0788414394d812526f434c26ecd95285cd793/backend/oqtopus_cloud/provider/routers/jobs.py#L54

Use "|" instead of union